py-polars/polars/series/series.py

```diff
@@ -4772,6 +4772,21 @@ def n_unique(self) -> int:
         """
         return self._s.n_unique()
 
+    def approx_unique(self) -> int:
+        """
+        Approx count unique values in this Series.
+
+        This is done using the HyperLogLog++ algorithm for cardinality estimation.
+
+        Examples
+        --------
+        >>> s = pl.Series("a", [1, 2, 2, 3])
+        >>> s.approx_unique()
+        3
+
+        """
+        return self._from_pyseries(self._s.approx_unique()).item()
```

Comment:
This should dispatch via the expression API.

Reply:
done